### PR TITLE
[ICP-13162] Add alarm event when tamper is triggered

### PR DESCRIPTION
**Pull request by MGoralczykS**
@greens @KKlimczukS @PKacprowiczS @ZWozniakS @BJanuszS @MWierzbinskaS 
Could you check this?

### CODE REVIEW

**Comment by greens** [comment]
You'd also need to clear the alarm/chime when tamper is cleared.

**Comment by MGoralczykS** [comment]
@greens Fixed

**Comment by greens** [comment]
My guess is most of the actual tamper clearing is done by `clearTamper` so you need to also send an alarm off there too, I'd imagine?

**Comment by MGoralczykS** [comment]
@greens Done

File: devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy [hunk 1]
```diff
@@ -252,7 +255,7 @@ def resetActiveSound() {
 def setActiveSound(soundId) {
 	String childDni = "${device.deviceNetworkId}:${soundId}"
 	def child = childDevices.find { it.deviceNetworkId == childDni }
-	child?.sendEvent(name: "chime", value: "on")
+	child?.sendEvent(name: "chime", value: "chime")
```
Comment (dkirker) [inline devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:263]: Nice catch! ;)

**Review by dkirker — DISMISSED** [review]

File: devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy (hunk 1 above)
Comment (MGoralczykS) [reply devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy:263]: Thanks! :)

**Comment by MGoralczykS** [comment]
@greens @dkirker I changed time in runIn and also fix for Ganesha's comment in ICP-13163. Could you check it?

**Review by dkirker — APPROVED** [review]

**Pull request merged**